parm/config/gfs/config.esfc

```diff
@@ -21,10 +21,20 @@ if [[ "${DO_JEDIATMENS}" == "YES" ]]; then
    export DONST="NO"
 fi
 
+_RUN=${RUN:-"gfs"}
+_RUN=${RUN/enkf/}
+
+if [[ "${_RUN}" == "gfs" ]]; then  
```

Comment:
```suggestion
if [[ "${RUN/enkf}" == "gfs" ]]; then  
```
Don't really need `_RUN`.

Reply:
@aerorahul @TravisElless-NOAA I thought we need the _RUN to turn off land DA for the enkf early cycle. Is that not right?

Reply:
Since `RUN` is already defined, and we are just using it to setup this if statement, using @aerorahul's suggestion of just doing the enkf optional chopping of `RUN` in the if statement call instead of defining the chopped RUN before should be sufficient.

Reply:
Thanks @TravisElless-NOAA I missed the /enkf in the if statement!